llmfoundry/models/mpt/configuration_mpt.py

```diff
@@ -208,7 +212,7 @@ def _validate_config(self):
         if self.init_config.get('name', None) is None:
             raise ValueError(f"{self.init_config=} 'name' needs to be set.")
         if not self.learned_pos_emb and not self.attn_config['alibi']:
-            raise warnings.warn(
+            warnings.warn(
```

Comment:
Having neither be set is a valid config of NoPE

Reply:
Yeah, if you choose to `raise` it leads to an error (so you can't actually use NoPE):

![image](https://github.com/mosaicml/llm-foundry/assets/15602962/e8e19312-e768-4a9f-91a3-4984b3ce6964)

Reply:
I'm an idiot

Reply:
haha naw all good